### ISSUE TITLE
Can you share how to make an iterm2 color profile given a certain palette?

### ISSUE THREAD

**Issue by svanharmelen** [sainnhe/dotfiles#9]
I love the Sonokai setup and use it for just about everything ❤️  For neovim I use a slightly adjusted version (see [here](https://github.com/sainnhe/sonokai/compare/master...svanharmelen:sonokai:master)) and I would like to port that version to an iterm2 profile so I can use it with Ghostty (it actually uses yet another format so if you already know that it would even be better, but it has logic to convert iterm2 profiles so I should be able to use that as well).

I noticed you have a few ported ones in here so I assume (hope 😊) you know which colors should be used for which items in the profile. Thanks for any help and/or pointers!!

**Comment by sainnhe** [comment]
Do you mean you want to port your modified version to iTerm2 but have no idea on which color should be used on a specific item?

You can import my sonokai iTerm2 theme file via settings -> Profiles -> Your Profile -> Colors -> Color Presets -> Scroll down and you'll see "Import" and "Export" button. Import my presets in `dotfiles/.macos/iTerm2/colors`, then activate it.

After that, you can change the colors to your modified version in the same setting page based on my preset. Finally, you can export the modified color preset to a new file.

**Comment by svanharmelen** [comment]
> Do you mean you want to port your modified version to iTerm2 but have no idea on which color should be used on a specific item?

Yes, that is exactly what I meant 😊 Thank you for the pointer I will give it a try tomorrow and let you know if it worked out!

**Comment by svanharmelen** [comment]
Worked like a charm. Thanks for the help ❤️
